src/utils/string/StringUtils.js

```diff
@@ -385,8 +384,7 @@ const getDurationTimeText = (selectedtime) => {
 };
 
 const getCaptionWithLinkStyle = (text) => {
-  const linkRegex = /(https?:\/\/\S+)+/g;
-  return reactStringReplace(text, linkRegex, (match) => (
+  return reactStringReplace(text, urlRegex, (match) => (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `reactStringReplace` function is used here to replace URLs in the text with a `HighlightText` component. However, it's not clear where the `onOpenLink` function is defined. Make sure it's imported or defined in this file.
```diff
- return reactStringReplace(text, urlRegex, (match) => (
+ return reactStringReplace(text, urlRegex, (match) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->